src/components/sections/About.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { SectionProps } from '../../utils/SectionProps';
+
+const propTypes = {
+  ...SectionProps.types
+}
+
+const defaultProps = {
+  ...SectionProps.defaults
+}
+
+
+const About = ({
+  className,
+  topOuterDivider,
+  bottomOuterDivider,
+  topDivider,
+  bottomDivider,
+  hasBgColor,
+  invertColor,
+  ...props
+}) => {
+
+  const [videoModalActive, setVideomodalactive] = useState(false);
+
+  const openModal = (e) => {
+    e.preventDefault();
+    setVideomodalactive(true);
+  }
+
+  const closeModal = (e) => {
+    e.preventDefault();
+    setVideomodalactive(false);
+  }   
+
+  const outerClasses = classNames(
+    'hero section center-content',
+    topOuterDivider && 'has-top-divider',
+    bottomOuterDivider && 'has-bottom-divider',
+    hasBgColor && 'has-bg-color',
+    invertColor && 'invert-color',
+    className
+  );
+
+  const innerClasses = classNames(
+    'hero-inner section-inner',
+    topDivider && 'has-top-divider',
+    bottomDivider && 'has-bottom-divider'
+  );
+
+  const ImgWithFallback = ({
+                             src,
+                             fallback,
+                             type = 'image/avif',
+                             ...delegated
+                           }) => {
+    return (
+        <picture>
+          <source srcSet={src} type={type} />
+          <img src={fallback} {...delegated} />
+        </picture>
+    );
+  };
+
+  return (
+    <section
+      {...props}
+      className={outerClasses}
+    >
+      <div className="container">
+        <div className={innerClasses}>
+          <div className="hero-content">
+            <h1 className="mt-0 mb-16 reveal-from-bottom" data-reveal-delay="200" >
+            Our Story
+            </h1>
+           
+            <div className="container-sm" style={{marginTop: "50px", fontWeight: "500", color:"#9CA9B3", lineHeight: "30px", fontSize: "20px", textAlign: "left"}}>
+              <p className="m-0 mb-32 reveal-from-bottom" data-reveal-delay="400">John, one of the founders of Shrub had always loved the financial markets and being able to research companies and invest in them. For years he kept up this hobby, keeping up with all of the financial news and then buying an ownership stake in companies he believed were undervalued.</p>
```

Comment:
and may be we remove all the data delays

Reply:
removed data delays by deleting data-reveal-delay="400"  from /about.js

Reply:
may need to clean this guy up as well, and then we are ready to go 🚀 `reveal-from-bottom`

Reply:
reveal-from-bottom removed